lib/matplotlib/quiver.py

```diff
@@ -720,10 +720,6 @@ def _h_arrows(self, length):
             np.copyto(Y, Y1, where=tooshort)
         # Mask handling is deferred to the caller, _make_verts.
         return X, Y
-
-    quiver_doc = _api.deprecated("3.7")(property(lambda self: _quiver_doc))
-
-
 _barbs_doc = r"""
```

Comment:
These two blank lines should be kept.
```suggestion


_barbs_doc = r"""
```